apollo-router/tests/integration/introspection.rs

```diff
@@ -107,51 +107,6 @@ async fn two_operations() {
     "###);
 }
 
-#[tokio::test]
-async fn operation_name_error() {
```

Comment:
Instead of removing these two tests can we change them to use the router service?